dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```
@@ -172,31 +172,31 @@ stopAppStartUponListenerException.name=Stop application for unhandled listener e
 stopAppStartUponListenerException.desc=Some web applications depend on context listeners for setup before the web application starts. When this property is set to true, the application stops starting up when an unhandled exception is thrown from the context listeners. For servlet-5.0 and newer Servlet features, the default value is true. 
 
 redirectToRelativeUrl.name=Send redirect response to a relative URL location
-redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location. 
+redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send a redirect response without converting the URL to an absolute location. 
 
 addStrictTransportSecurityHeader.name=Add the HTTP strict transport security header
 addStrictTransportSecurityHeader.desc=Enables the HTTP Strict Transport Security (HSTS) header for HTTPS responses and allows a value to be set for that header, for example: "max-age=31536000;includeSubDomains". HSTS can also be configured in the web.xml file by setting the "com.ibm.ws.webcontainer.ADD_STS_HEADER_WEBAPP" context parameter.
 
 allowQueryParamWithNoEqual.name=Allow query parameters with no value
-allowQueryParamWithNoEqual.desc=When this property is set to true, if the query parameter in a URL contains only the string name, the server returns an empty string as the value for the request.getParameter(name) query. When this property is set to false, the server returns null as the value for the request.getParameter(name) query. The default value is false, but is true for servlet 5.0 and higher.
+allowQueryParamWithNoEqual.desc=When this property is set to true, the server returns an empty string as the value for the request.getParameter(name) query if the query parameter in the URL contains only the string name. When this property is set to false, the server returns null as the value for the request.getParameter(name) query. In servlet-5.0 and later, the default value is true. In servlet-4.0 and earlier, the default is false.
```

```suggestion
allowQueryParamWithNoEqual.desc=When this property is set to true, if the query parameter in a URL contains only the string name, the server returns an empty string as the value for the request.getParameter(name) query. When this property is set to false, the server returns null as the value for the request.getParameter(name) query. In servlet-5.0 and later, the default value is true. In servlet-4.0 and earlier, the default is false.
```

---

@dmuelle  which do you prefer?  I'm leaning on current wording (if is at the end rather than in the middle of the sentence)

---

@volosied - does `if the query parameter in a URL contains only the string name` also apply if the prop is set to false? If so, if might be more clear to first set that condition in the description... something like

```suggestion
allowQueryParamWithNoEqual.desc=This property determines how the server responds when the query parameter in a URL contains only the string name. When it is set to true, the server returns an empty string as the value for the request.getParameter(name) query.  When it is set to false, the server returns null as the value for the request.getParameter(name) query. In servlet-5.0 and later, the default value is true. In servlet-4.0 and earlier, the default is false.
```